modules/persistent-storage-csi-drivers-supported.adoc

```diff
@@ -42,6 +42,10 @@ The following table describes the CSI drivers that are installed with {product-t
 * Offline volume expansion: minimum required vSphere version is 6.7 Update 3 P06
 
 * Online volume expansion: minimum required vSphere version is 7.0 Update 2.
+
+3.
+
+* Does not support offline snapshots or resize, volume must be attached to a running pod.
```

Comment:
Small nit. This should be two sentences: "Does not support offline snapshots or resize. Volume must be attached to a running pod."

Reply:
Fixed, thanks!